### DISCUSSION TITLE
Show clients count

### DISCUSSION THREAD

**Discussion by mitrios-hub** [wg-easy/wg-easy#370] [Ideas]
Hello! Would it be possible to add clients count? IMHO it's useful.
I changed line 60, but it will be better to change in branch:
`<p class="text-1xl font-medium">Clients: {{clients.length}}</p>`

**Comment by nevalashka** [comment]
> Hello! Would it be possible to add clients count? IMHO it's useful.
> I changed line 60, but it will be better to change in branch:
> `<p class="text-1xl font-medium">Clients: {{clients.length}}</p>`

Useful man, thanks!
